### PR TITLE
Cyborgs Extended

### PR DESCRIPTION
**Pull request by OCOtheOmega**
# Description

# TODO

# Media

# Changelog

Yeah its simple but look good, everything tested, no errors.

изображение

### CODE REVIEW

File: Resources/Prototypes/SimpleStation14/Entities/Mobs/Player/borgs.yml
Comment (DEATHB4DEFEAT) [inline Resources/Prototypes/SimpleStation14/Entities/Mobs/Player/borgs.yml]: Dont know what the base looks like, but only the sprite path looks like it needs to be redefined, everything else would be on the parent

File: Resources/Prototypes/SimpleStation14/Entities/Mobs/Player/borgs.yml
Comment (DEATHB4DEFEAT) [inline Resources/Prototypes/SimpleStation14/Entities/Mobs/Player/borgs.yml]: also probably redundant

File: Resources/Prototypes/SimpleStation14/Entities/Mobs/Player/borgs.yml
Comment (DEATHB4DEFEAT) [inline Resources/Prototypes/SimpleStation14/Entities/Mobs/Player/borgs.yml]: redundant

wish the linter checked for these

File: Resources/Prototypes/SimpleStation14/Entities/Mobs/Player/borgs.yml
Comment (DEATHB4DEFEAT) [inline Resources/Prototypes/SimpleStation14/Entities/Mobs/Player/borgs.yml]: combat mode?

File: Resources/Prototypes/SimpleStation14/Entities/Mobs/Player/borgs.yml
Comment (DEATHB4DEFEAT) [inline Resources/Prototypes/SimpleStation14/Entities/Mobs/Player/borgs.yml]: unneeded comment

File: Resources/Prototypes/SimpleStation14/Entities/Mobs/Player/borgs.yml
Comment (DEATHB4DEFEAT) [inline Resources/Prototypes/SimpleStation14/Entities/Mobs/Player/borgs.yml]: probably don't need this and combat mode, I'll let psp decide

File: Resources/Prototypes/SimpleStation14/Entities/Mobs/Player/borgs.yml
Comment (DEATHB4DEFEAT) [inline Resources/Prototypes/SimpleStation14/Entities/Mobs/Player/borgs.yml]: some sort of drink holders would be better so it can interact with the bar machinery instead of being one time use

Comment (DEATHB4DEFEAT) [reply Resources/Prototypes/SimpleStation14/Entities/Mobs/Player/borgs.yml]: look at the mediborg

File: Resources/Prototypes/SimpleStation14/Roles/Jobs/Cyborg/clown_cyborg.yml
Comment (DEATHB4DEFEAT) [inline Resources/Prototypes/SimpleStation14/Roles/Jobs/Cyborg/clown_cyborg.yml]: clown playtime too

File: Resources/Prototypes/SimpleStation14/Roles/Jobs/Cyborg/engineer_cyborg.yml
Comment (DEATHB4DEFEAT) [inline Resources/Prototypes/SimpleStation14/Roles/Jobs/Cyborg/engineer_cyborg.yml]: engineer playtime too

File: Resources/Prototypes/SimpleStation14/Roles/Jobs/Cyborg/janitor_cyborg.yml
Comment (DEATHB4DEFEAT) [inline Resources/Prototypes/SimpleStation14/Roles/Jobs/Cyborg/janitor_cyborg.yml]: janitor time too

File: Resources/Prototypes/SimpleStation14/Roles/Jobs/Cyborg/security_cyborg.yml
Comment (DEATHB4DEFEAT) [inline Resources/Prototypes/SimpleStation14/Roles/Jobs/Cyborg/security_cyborg.yml]: security time too

File: Resources/Prototypes/SimpleStation14/Roles/Jobs/Cyborg/service_cyborg.yml
Comment (DEATHB4DEFEAT) [inline Resources/Prototypes/SimpleStation14/Roles/Jobs/Cyborg/service_cyborg.yml]: bartender time too

File: Resources/Prototypes/SimpleStation14/Entities/Mobs/Player/borgs.yml
Comment (Pspritechologist) [inline Resources/Prototypes/SimpleStation14/Entities/Mobs/Player/borgs.yml:323]: I wouldn't want situations of people thinking the Clown has authorized something has legitimate chicanery, but it turns out it was just some lowly Cyborg.

**Review by DEATHB4DEFEAT — CHANGES_REQUESTED** [review]
lots of redundant redefinitions

File: Resources/Prototypes/SimpleStation14/Entities/Mobs/Player/borgs.yml
Comment (Pspritechologist) [inline Resources/Prototypes/SimpleStation14/Entities/Mobs/Player/borgs.yml]: This feels like a bad idea :P
Those things are so noisy, and only serve to pester Admins. Linking it to someone's very being feels unnecessary.

File: Resources/Prototypes/SimpleStation14/Entities/Mobs/Player/borgs.yml
Comment (Pspritechologist) [inline Resources/Prototypes/SimpleStation14/Entities/Mobs/Player/borgs.yml]: Needs Clown laws. I might write some up.
These really should also be localized, but I assume the code doesn't support that. Lame.

File: Resources/Prototypes/SimpleStation14/Entities/Mobs/Player/borgs.yml
Comment (Pspritechologist) [inline Resources/Prototypes/SimpleStation14/Entities/Mobs/Player/borgs.yml]: No pneumatic launcher for pies??

File: Resources/Prototypes/SimpleStation14/Entities/Mobs/Player/borgs.yml
Comment (Pspritechologist) [inline Resources/Prototypes/SimpleStation14/Entities/Mobs/Player/borgs.yml:218]: So useless, I love it :P

File: Resources/Prototypes/SimpleStation14/Entities/Mobs/Player/borgs.yml
Comment (Pspritechologist) [inline Resources/Prototypes/SimpleStation14/Entities/Mobs/Player/borgs.yml]: Needs Jani laws

File: Resources/Prototypes/SimpleStation14/Entities/Mobs/Player/borgs.yml
Comment (Pspritechologist) [inline Resources/Prototypes/SimpleStation14/Entities/Mobs/Player/borgs.yml:211]: This one *needs* to be made into a solution container, ala a mop bucket, so that they can store puddles in themselves. Then, people can shove them over!

File: Resources/Prototypes/SimpleStation14/Entities/Mobs/Player/borgs.yml
Comment (Pspritechologist) [inline Resources/Prototypes/SimpleStation14/Entities/Mobs/Player/borgs.yml]: 😨

File: Resources/Prototypes/SimpleStation14/Entities/Mobs/Player/borgs.yml
Comment (Pspritechologist) [inline Resources/Prototypes/SimpleStation14/Entities/Mobs/Player/borgs.yml]: Needs mining laws, I guess? This is a weird Cyborg :P

File: Resources/Prototypes/SimpleStation14/Entities/Mobs/Player/borgs.yml
Comment (Pspritechologist) [inline Resources/Prototypes/SimpleStation14/Entities/Mobs/Player/borgs.yml:63]: Can these be charged??

File: Resources/Prototypes/SimpleStation14/Entities/Mobs/Player/borgs.yml
Comment (Pspritechologist) [inline Resources/Prototypes/SimpleStation14/Entities/Mobs/Player/borgs.yml:65]: Doesn't this burn out??

File: Resources/Prototypes/SimpleStation14/Entities/Mobs/Player/borgs.yml
Comment (Pspritechologist) [inline Resources/Prototypes/SimpleStation14/Entities/Mobs/Player/borgs.yml:66]: CAN THIS BE APPLIED TO PEOPLE??

File: Resources/Prototypes/Maps/parkstation.yml
Comment (Pspritechologist) [inline Resources/Prototypes/Maps/parkstation.yml]: This is _so many_ of them :P

File: Resources/Prototypes/SimpleStation14/Roles/Jobs/Cyborg/clown_cyborg.yml
Comment (Pspritechologist) [inline Resources/Prototypes/SimpleStation14/Roles/Jobs/Cyborg/clown_cyborg.yml:8]: These should need specific playtime in their respective jobs/as their respective roles. Goes for all of them.

**Review by Pspritechologist — CHANGES_REQUESTED** [review]
This is pretty dang cool, but I am concerned that it only increases the severity of the existing issue with Cyborgs.
The Janitor Cyborg in particular effectively entirely replaces the Janitor themselves in terms of usefulness, they're stronger, better at doing their job, etc.. Cyborgs are fine in a limited extent, but these ones I worry will significantly displace existing jobs.

File: Resources/Prototypes/SimpleStation14/Entities/Mobs/Player/borgs.yml
Comment (OCOtheOmega) [reply Resources/Prototypes/SimpleStation14/Entities/Mobs/Player/borgs.yml:66]: Yeah it can stun and hadcuff target and they dont lost handcuffs

File: Resources/Prototypes/SimpleStation14/Entities/Mobs/Player/borgs.yml
Comment (OCOtheOmega) [reply Resources/Prototypes/SimpleStation14/Entities/Mobs/Player/borgs.yml]: Check honk laws

File: Resources/Locale/en-US/SimpleStation14/Prototypes/job/job-names.ftl
Comment (DEATHB4DEFEAT) [inline Resources/Locale/en-US/SimpleStation14/Prototypes/job/job-names.ftl]:
```suggestion
job-name-shaftminer-cyborg = mining cyborg
```

File: Resources/Prototypes/SimpleStation14/Entities/Mobs/Player/borgs.yml
Comment (DEATHB4DEFEAT) [inline Resources/Prototypes/SimpleStation14/Entities/Mobs/Player/borgs.yml]: server-side sprite component was removed, netsync no longer exists

File: Resources/Prototypes/SimpleStation14/Entities/Mobs/Player/borgs.yml
Comment (DEATHB4DEFEAT) [inline Resources/Prototypes/SimpleStation14/Entities/Mobs/Player/borgs.yml]: redefinition

**Review by DEATHB4DEFEAT — CHANGES_REQUESTED** [review]

**Comment by OCOtheOmega** [comment]
Yep it need add after IPC merge
Make cyborgs more special
For future need add charge to sec cyborg tools but it IPC moment

**Comment by OCOtheOmega** [comment]
Well wiz start make own cyborgs, soo this usless.

**Pull request closed without merging**
